frontend/src/app/associations/association-members/association-members.component.ts

```diff
@@ -31,13 +33,29 @@ export class AssociationMembersComponent implements OnInit {
 
     ngOnInit() {
         const id = this.route.snapshot.paramMap.get('id');
-        this.api.get("associations/" + id + "/").subscribe(
-            association => this.association = association,
-            error => this.error = error.message
-        );
 
         this.api.get('users/').subscribe(
-            users => this.users = users,
+            users => {
+                let users_index = {};
```

Comment:
Not a big fan of all that because the users request is gonna be huge as it gets all data from all users. We will fetch way too much data for what will be used. Second I don't see where association.members is used (at least in its updated form), can you point to it?

I would populate the association field directly from the api request without changing members. If there is not enough information in what is returned from the API then change the API.
Second I would fetch only the list of usernames and use that in the ng-select element.